assets/src/scss/new-identity/style.scss

```diff
@@ -42,4 +42,8 @@
   --button-secondary--hover--background: var(--p4-dark-green-800);
   --button-secondary--hover--border-color: var(--p4-dark-green-800);
   --button-secondary--hover--color: var(--white);
+  --link--color: var(--grey-900);
+  --link--visited--color: var(--grey-900);
+  --link--hover--color: var(--grey-900);
+  --link--hover--text-decoration: none;
```

Comment:
This line `--link--hover--text-decoration: none;` also affect contextual links (such as the Articles block's tags) that actually should still have the underline on hover, so I think we need to create a variable for those